packages/core/src/utils/validator.ts

```diff
@@ -17,11 +17,11 @@ export const ibmOpenapiValidator = async (specs: OpenAPIObject) => {
   spectral.setRuleset(ibmOpenapiRuleset);
   const { errors, warnings } = await spectral.run(specs);
 
-  if (warnings.length) {
+  if (warnings && warnings.length) {
     ibmOpenapiValidatorWarnings(warnings);
   }
 
-  if (errors.length) {
+  if (errors && errors.length) {
```

Comment:
I'm curious why this was running undefined in the first place -- is that normal, or is it because `spectral.run(...)` failed?

Reply:
It is not for me i have no issues with Orval so I am not sure if its related to your environment or settings or something?